contracts/transfer/circuits/src/execute/crossover.rs

```diff
@@ -0,0 +1,60 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) DUSK NETWORK. All rights reserved.
+
+use dusk_plonk::jubjub::JubJubExtended;
+use dusk_plonk::prelude::*;
+
+#[derive(Debug, Default, Clone)]
+pub struct CircuitCrossover {
+    value_commitment: JubJubExtended,
+    value: u64,
+    blinding_factor: JubJubScalar,
```

Comment:
Should we also not implement a `Default` here?

Reply:
`Default` is already implemented via derive macro